frontend/src/components/App/App.tsx

```diff
@@ -82,13 +82,9 @@ const App = () => {
                         {/* Experiment */}
                         <Route path={URLS.experiment} element={<Experiment />} />
 
-                        {/* Session - ⚠️ What is the purpose of this non-functional route? */}
-                        <Route path={URLS.session} />
```

Comment:
I removed this route because `URLS.session` does not exist in the `URLS` object. (It does exist in the `URLS` object in `API.ts`, but I assume that's something different.)